lib/validators/liveValidator.js

```diff
@@ -174,7 +174,8 @@ class LiveValidator {
           return Promise.resolve(self.cache);
         }).catch(function (err) {
           log.debug(`Unable to initialize "${swaggerPath}" file from SpecValidator. Error: ${err}`);
-          return Promise.reject(err);
+          // Do Not reject promise in case, we cannot initialize one of the swagger
```

Comment:
minor: you may want to expand the comment to indicate, that therefore we're ignoring this swagger file, logging and continuing building the cache with the rest of the valid ones.
Also, not sure if you want to keep the commented code in the next line of just leave a good comment on what the "catch" is expected to do an why.